src/chapters/interfaces-and-polymorphism/given-interfaces.rst

```diff
@@ -0,0 +1,114 @@
+Interfaces In The Wild
+======================
+
+The first situations where you’ll want to use interfaces involve applying pre-defined interfaces 
+and classes that are part of C#. Here are a few examples.
+
+IComparer<T>
+------------
+
+**Purpose**: A class implements this interface to compare two objects of a given class. This is 
+a *parameterized* interface, which means that when using it, you need to specify the class that 
+it will be comparing. For example, ``IComparer<Job>`` would compare ``Job`` objects.
+
+**Important Methods and Properties**: ``Compare(T, T)``
+
+`IComparer.Compare
+Documentation <https://msdn.microsoft.com/en-us/library/system.collections.icomparer.compare(v=vs.110).aspx>`__
+
+This interface can be used to determine, given two objects of the given
+type, which one is *greater* than the other. It is also used by
+collections such as
+`List <https://msdn.microsoft.com/en-us/library/6sh2ey19(v=vs.110).aspx>`__
+to sort its contents with the
+`Sort <https://msdn.microsoft.com/en-us/library/234b841s(v=vs.110).aspx>`__
+method.
+
+IEnumerable<T>
+--------------
+
+**Purpose**: Enable iteration over a collection of objects using
+``foreach``.
```

Comment:
I think it would be helpful here and elsewhere on this page to acknowledge that students may have already done something similar (in this case, they have already used ``foreach``) and why interfaces make it the same/different? Are these interfaces that students were accessing this whole time? If so, how did they access these interfaces? I think some students will need additional clarity.